wiki/Tournaments/VAF/1/en.md

```diff
@@ -0,0 +1,691 @@
+13. Disrespect of the assigned referee, and/or opponent, will not be tolerated. (Applies towards other staff members in the Discord server environment)
+14. If a pick has been tied for 2 times after replaying it, the pick will be nullified.
+15. The mappool size will follow the listed format:
+    - Swiss bracket: 5 Nomod, 3 Hidden, 3 HardRock, 3 DoubleTime, 3 FreeMod, No Tiebreaker
+    - Semifinals, Finals, and Grand Finals: 5 Nomod, 3 Hidden, 3 HardRock, 3 DoubleTime, 3 FreeMod, 1 Tiebreaker
+16. Any members of staff apart from **Organiser, Mappool selector, and Referee are allowed to play**.
+17. All match schedules will be written in UTC.
+18. Reschedules will always be taken as long as they are **24 hours before** the first schedule of the running stage.
+19. Mappools will be released by each week's Monday so every match can occur before that. This applies both for Swiss bracket and during the knockout stage.
+
+### Registration
+
+1. Everyone that meets the rank restricted requirements is able to sign up. Shen registration is coming to a close, players are allowed to bypass the rank restriction, but if the player isn't within the rank range by the time registration ends their registration will be removed.
+2. To register, each player must follow the format posting: Their osu! user ID followed by timezone (in UTC) in the `#registration` text-channel in the VAF Discord server, which will only be open for 2 weeks until the registrations end.
+3. Signing up does not guarantee participation in the tournament. Players will have to stay within the cutoff until the registrations close. The cutoff will be held at 64 players.
+
+### Swiss style
+
+1. Swiss style is a league style format which consists in playing against an x amount of oponents depending on the format and the rounds existing. This implies that each player will not play against everyone in the swiss. Each matchup will depend on the seeding and performance of every player in the swiss bracket. There is an exception where in case of a tie a player might rematch someone depending on their point difference (this depends on the Challonge bracket). More explanation on video format can be accessed through the [following video](https://youtu.be/WsvFrxJPBZA).
+2. Each player will play throughout 6 weeks, 6 matches/rounds of swiss style.
+3. Matchups will be paired between players who posses the same amount of W-L-D (for example someone that goes 3-0 will be prioritized against someone who also is 3-0). What is more, playing against the same player twice will be avoided unless it's completely necessary.
+4. Every match will use BO13 format, this means throughout the entire swiss bracket a player will need 7 points to win a match or 6 to guarantee a tie. Semifinals onwards will also use BO13 format. The main difference is swiss bracket doesn't have a tiebreaker.
+5. In case of a tie in W-L-D at the end of the 6 rounds/matches several tiebreaker methods will be used on different information:
```

Comment:
```suggestion
5. In case of a tie in W-L-D at the end of the 6 rounds/matches, several tiebreaker methods will be used depending on different information:
```